source/beamline/BioXAS/BioXASSideMonochromator.cpp

```diff
@@ -45,7 +45,7 @@ BioXASSideMonochromator::BioXASSideMonochromator(QObject *parent) :
 	region_->setPaddleStatusControl(paddleStatus_);
 	region_->setKeyStatusControl(keyStatus_);
 	region_->setBrakeStatusControl(brakeStatus_);
-	region_->setBraggControl(bragg_);
+	region_->setBraggControl(stepBragg_);
```

Comment:
Previously, the region control was using the encoder-based bragg control, and this change corrects it to using the step-based control. I wanted to make sure it was consistent with the new preferred option.